README.md

```diff
@@ -1,73 +1,70 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Getting Started with Create React App.
 
-# Project Name
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-> Description the project.
+## Available Scripts
 
+In the project directory, you can run:
 
-## Built With
+### `npm start`
 
-- Major languages
-- Frameworks
-- Technologies used
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
 
-## Live Demo (if available)
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
 
-[Live Demo Link](https://livedemo.com)
+### `npm test`
 
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
 
```

Comment:
- [ ] Kindly Add a descriptive README using the [Microverse README template](https://github.com/microverseinc/readme-template) and include all the necessary sections (project title, description of the project, author, etc...) that are available instead of using the generated README from CRA.